blog/_posts/2017-08-01-one-month-of-RGSoC.md

```diff
@@ -0,0 +1,42 @@
+---
+title: One month of RGSoC
+layout: post
+created_at: Tue Aug 01 2017
+permalink: blog/2017-08-01-one-month-of-RGSoC
+current: blog
+author: Vaishali Thakkar
+twitter: RailsGirlsSoC
+categories:
+- news
+---
+
+This year’s program has started off with a bang! As today marks the start of the fourth week of RGSoC, we thought you would be interested in seeing what our amazing teams have been doing since the first day of our program.
+
+From enjoying Global Kick-off events to having their kick-off calls with team members, from making their first pull request to preparing for conferences, our students are ready to mark their presence in the open source world.
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Ah that satisfaction when you run the first module on Tessel ❤️ Ambient module measuring light and sound levels <a href="https://twitter.com/tesselproject">@tesselproject</a> <a href="https://twitter.com/hashtag/rgsoc?src=hash">#rgsoc</a> <a href="https://t.co/NTFyQzOeaJ">pic.twitter.com/NTFyQzOeaJ</a></p>&mdash; 276linesofCode (@276linesofCode) <a href="https://twitter.com/276linesofCode/status/885099913998217222">July 12, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Weekly sync with the whole <a href="https://twitter.com/RailsGirlsSoC">@RailsGirlsSoC</a> <a href="https://twitter.com/ifmeorg">@ifmeorg</a> <a href="https://twitter.com/sophierose239">@sophierose239</a> <a href="https://twitter.com/HashNotAdam">@HashNotAdam</a> <a href="https://twitter.com/JennyNamster">@JennyNamster</a> <a href="https://twitter.com/fleurchild">@fleurchild</a> <a href="https://twitter.com/tmoore">@tmoore</a> <a href="https://twitter.com/adelsmee">@adelsmee</a> <a href="https://t.co/vQNohKAiyT">pic.twitter.com/vQNohKAiyT</a></p>&mdash; ifPairElseUnknown (@IfPairElse) <a href="https://twitter.com/IfPairElse/status/884596784349065217">July 11, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">It was so nice to meet our incredible supervisor <a href="https://twitter.com/ines_opcoelho">@ines_opcoelho</a> !! We had a meeting today with her and all our team :D <a href="https://t.co/Ep0gpb3DYS">https://t.co/Ep0gpb3DYS</a></p>&mdash; Rails Gyn (@RailsGyn) <a href="https://twitter.com/RailsGyn/status/882023345478414336">July 3, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+*Our teams are having fun with kick-off calls & their projects [Credit: RGSoC Teams]*
```

Comment:
Have you tried using the `image-credits` style on these captions as well? I think it would be good to have this consistent — but maybe it doesn't look good? 
Also, in the name of consistency, could you use round parentheses here `( )` instead of square brackets `[ ]`? Thanks!